citi-onboarding/Aqua-Jr
# feature/call-to-blog

## What I did
- Created the call to blog component

## How to test
1. Access the branch through your terminal:
```
   git checkout develop
   git pull
   git checkout feature/call-to-blog
```
4. Access the client folder in another terminal window and start React:
```
   cd /client
   yarn start
```
5. In your browser, go to:
``` 
   localhost:3000
```

6. Check if it only appears when the screen is less wider then 1025px
7. Check if it is just like the mockup
